packages/x-data-grid-premium/src/hooks/features/sidebar/gridSidebarInterfaces.ts

```diff
@@ -0,0 +1,17 @@
+export enum GridSidebarValue {
+  pivot = 'pivot',
+}
```

Comment:
We don't have an enum casing convention :| but `Enum.field` looks like the least common option, we use `Enum.Field` more commonly, or `Enum.FIELD` in a close second place.